src/platform_impl/macos/window.rs

```diff
@@ -1094,6 +1117,28 @@ impl WindowExtMacOS for UnownedWindow {
             }
         }
     }
+
+    #[inline]
+    fn has_shadow(&self) -> bool {
+        unsafe {
+            if self.ns_window.hasShadow() == YES {
```

Comment:
You don't need to use `if` here; the body of this function could instead be:
```rust
unsafe { self.ns_window.hasShadow() } == YES
```

Reply:
done!